### ISSUE TITLE
syntax error reports module not found

### ISSUE THREAD

**Issue by jeremygaither** [wata727/tflint#385]
An argument used twice on a module causes the module loader to report the module as not found, returning the following misleading error:

```
Error: Failed to load configurations: main.tf:37,1-26: `linux_buildnodes` module is not found. Did you run `terraform init`?; , and 4 other diagnostic(s)
```

Running with debugging reveals the true error:

```
14:35:47 loader.go:132: [ERROR] Failed to load modules using the v0.11.0 ~ v0.11.7 module walker; Trying the v0.12 module walker...
14:35:47 loader.go:133: [DEBUG] Original error: .terraform/modules/9f6e691374b98e65401f06aa1836fc99/main.tf:29,3-11: Attribute redefined; The argument "max_size" was already set at .terraform/modules/9f6e691374b98e65401f06aa1836fc99/main.tf:27,3-11. Each argument may be set only once., and 7 other diagnostic(s)
...
Error: Failed to load configurations: main.tf:37,1-26: `linux_buildnodes` module is not found. Did you run `terraform init`?; , and 4 other diagnostic(s)
```

TFLint version 0.9.3

**Comment by wata727** [comment]
Are you using Terraform v0.11.x? Or have you run `terraform init` with v0.11.x?
Perhaps this is because TFLint has partial support for v0.11.x. The syntax error was caused by trying to load a module in the v0.11.x way.

Since the v0.12.x module does not exist, the "module is not found" error is the correct error. Please check if the same error occurs by `terraform init` with terraform v0.12.x.

In the near future, I plan to fix this problem by completely removing support for v0.11.x.

**Comment by jeremygaither** [comment]
Yes, this is with Terraform v0.11.14, init ran the same way. The modules all pass validate as well, and I've applied the module several times. This is not a module meant for 0.12 yet, so there is no 0.12 version of the module. The actual module is a subfolder.

Should I revert to an earlier version of tflint for my projects still using Terraform 0.11?

**Comment by jeremygaither** [comment]
I suppose I was most confused by the standard error message, when a much different error was shown in the debug log. Is it failing through because it assumes invalid 0.11 syntax might be valid 0.12 syntax, and therefore ignores the error and attempts to load it using the 0.12 loader?

**Comment by wata727** [comment]
> Should I revert to an earlier version of tflint for my projects still using Terraform 0.11?

Yes, to work with 0.11.x, you must use v0.7.6, but unfortunately, there are some bugs in this version...
Since it is difficult to support both 0.11 and 0.12 syntax, currently only 0.12 is supported.

> Is it failing through because it assumes invalid 0.11 syntax might be valid 0.12 syntax, and therefore ignores the error and attempts to load it using the 0.12 loader?

That's right. There will be room for improvement, but it maybe would be sufficient to simply remove the implementation other than the 0.12 loader.

**Comment by wonderg** [comment]
it would be great if tflint will support .tfswitchrc files to determine which terraform version should be used for verification
